scripts/bump_deps.lua

```diff
@@ -178,290 +106,106 @@ local function write_cmakelists_line(symbol, kind, value)
-  local commit_sha = get_gh_commit_sha(dependency.repo, commit)
-  if commit_sha ~= commit then
-    p('Not a commit: ' .. commit .. '. Did you mean version?')
-    die()
-  end
-  local archive = get_archive_info(dependency.repo, commit)
-  if dependency_name == 'Luv' then
-    warn_luv_symbol()
-  end
-  update_cmakelists(dependency, archive, short_commit(commit))
-end
-
-function M.version(dependency_name, version)
-  vim.validate('dependency_name', dependency_name, 'string')
-  vim.validate('version', version, 'string')
-  local dependency = assert(get_dependency(dependency_name))
-  verify_cmakelists_committed()
-  local commit_sha = get_gh_commit_sha(dependency.repo, version)
-  if commit_sha == version then
-    p('Not a version: ' .. version .. '. Did you mean commit?')
-    die()
-  end
-  local archive = get_archive_info(dependency.repo, version)
-  if dependency_name == 'Luv' then
-    write_cmakelists_line(dependency.symbol, 'VERSION', version)
-  end
-  update_cmakelists(dependency, archive, version)
-end
-
-function M.head(dependency_name)
-  local dependency = assert(get_dependency(dependency_name))
-  verify_cmakelists_committed()
-  local commit_sha = get_gh_commit_sha(dependency.repo, 'HEAD')
-  local archive = get_archive_info(dependency.repo, commit_sha)
-  if dependency_name == 'Luv' then
-    warn_luv_symbol()
-  end
-  update_cmakelists(dependency, archive, 'HEAD - ' .. short_commit(commit_sha))
-end
-
-function M.create_branch(dep)
-  explicit_create_branch(dep)
-end
-
-function M.submit_pr()
-  require_executable('git')
-
-  verify_branch('deps')
-
-  local nvim_remote = find_git_remote(nil)
-  local relevant_commit = assert(run_die({
-    'git',
-    'log',
-    '--grep=' .. commit_prefix,
-    '--reverse',
-    "--format='%s'",
-    nvim_remote .. '/master..HEAD',
-    '-1',
-  }, 'Failed to fetch commits'))
-
-  local pr_title
-  local pr_body
-
-  if relevant_commit == '' then
-    pr_title = commit_prefix .. 'bump some dependencies'
-    pr_body = 'bump some dependencies'
-  else
-    relevant_commit = relevant_commit:gsub("'", '')
-    pr_title = relevant_commit
-    pr_body = relevant_commit:gsub(commit_prefix:gsub('%(', '%%('):gsub('%)', '%%)'), '')
-  end
-  pr_body = pr_body .. '\n\n(add explanations if needed)'
-  p(pr_title .. '\n' .. pr_body .. '\n')
-  create_pr(pr_title, pr_body)
-end
-
 local function usage()
-  local this_script = _G.arg[0]:match('[^/]*.lua$')
-  print(([=[
+  local this_script = vim.fs.basename(_G.arg[0])
+  local help = ([=[
     Bump Nvim dependencies
 
     Usage:  nvim -l %s [options]
         Bump to HEAD, tagged version, commit, or branch:
-            nvim -l %s --dep Luv --head
-            nvim -l %s --dep Luv --version 1.43.0-0
-            nvim -l %s --dep Luv --commit abc123
-            nvim -l %s --dep Luv --branch
-        Create a PR:
-            nvim -l %s --pr
+            nvim -l %s luv --head
+            nvim -l %s luv --ref 1.43.0-0
```

Comment:
```suggestion
            %s luv --ref 1.43.0-0
```
And so on (yay shebang!)